src/Engine/RadiumEngine.hpp

```diff
@@ -89,8 +89,9 @@ class RA_ENGINE_API RadiumEngine {
      * @deprecated Will be removed from this class in the next release. A Mesh manager, that could
      * serve mesh by name will be implemented.
      */
-    [[deprecated]] Mesh* getMesh( const std::string& entityName, const std::string& componentName,
-                                  const std::string& roName = std::string() ) const;
+    [[deprecated]] Displayable* getMesh( const std::string& entityName,
+                                         const std::string& componentName,
+                                         const std::string& roName = std::string() ) const;

```

Comment:
I don't want to update it to avoid side-effect on older plugins, and anyway the method will be removed in v2.
Also, what's wrong with the comment ? It says it is deprecated and will be removed, right ?

Reply:
ok